testdata/sim-data/read_structure_manifest.csv

```diff
@@ -0,0 +1,4 @@
+fastq,read_structure
+/Users/swhite/projects/SINGULAR/nf-core-modules/fqtk_inputs/fastqs/subsets/Undetermined_S0_L001_I1_001_subset.fastq.gz,8B
```

Comment:
Should these be relative paths, or http links? I think it definitely should have the `/Users/swhite`.

Reply:
Error with the rebase I believe... definitely not intended to be local paths!

Reply:
@Emiller88 fixed in latest commit

Reply:
Okay, still not sure how this is going to work but we'll go with it for now.